omod/src/main/webapp/resources/scripts/services/privilegeService.js

```diff
@@ -0,0 +1,38 @@
+angular.module('privilegeService', ['ngResource', 'uicommons.common'])
+	.config(function ($httpProvider) {
+	})
+    .factory('Privilege', function($resource) {
+        return $resource("/" + OPENMRS_CONTEXT_PATH  + "/ws/rest/v1/privilege/:uuid", {
+            uuid: '@uuid'
+        },{
+            query: { method:'GET' }     // override query method to specify that it isn't an array that is returned
+        });
+    })
+    .factory('PrivilegeService', function(Privilege) {
+
+        return {
+
+            /**
+             * Fetches Privilege
+             *
+             * @param params to search against
+             * @returns $promise of array of matching Privileges (REST ref representation by default)
+             */
+            getPrivileges: function(params) {
```

Comment:
Do you realize that you actually never use functions anywhere?  I would just get rid of this factory entirely, I will clean up encountertTypeService.js because i know it mislead you guys. Sorry about that!